l10n_ch_scan_bvr/wizard/scan_bvr.py

```diff
@@ -158,6 +158,9 @@ def _create_invoice_line(self, data):
             specs = invoice_line_model._onchange_spec()
             default_values = invoice_line_model.default_get(specs)
             invoice_line_vals = specs.copy()
+            for val in invoice_line_vals:
+                if invoice_line_vals[val]:
+                    invoice_line_vals[val] = None
```

Comment:
I don't get why you empty all values of this dictionary.

If you need an empty dict you can do:

``` python
invoice_line_vals = {k: None for k in specs.keys()}
```

Reply:
You are right, thanks for the comment.
For the record, I need an empty dict because otherwise the values are not modified correctly in the onchange(): Some values are not updated there and receive the value '1', which is not correct.